addons/medical_gui/functions/fnc_updateCategories.sqf

```diff
@@ -21,8 +21,9 @@ params ["_display"];
     _x params ["_idc", "_category"];
 
     private _ctrl = _display displayCtrl _idc;
-    private _enable = GVAR(actions) findIf {_category == _x select 1 && {call (_x select 2)}} > -1;
-    if (_category isEqualTo "triage") then {_enable = true};
+    private _enable = if (_category == "triage") then { true } else {
+        GVAR(actions) findIf {_category == _x select 1 && {call (_x select 2)}} > -1
```

Comment:
```suggestion
    private _enable = if (_category isEqualTo "triage") then { true } else {
        GVAR(actions) findIf {_category isEqualTo (_x select 1) && {call (_x select 2)}} > -1
```

Reply:
Why? This should NOT be case-sensitive!

Reply:
I figured if it's a non-public fnc ACE uses internally, it wouldn't need to be insensitive.